app/views/admin/definitions/show.html.erb

```diff
@@ -149,7 +149,7 @@
          </div> <!-- END collapseTen -->
        </section>
      </div> <!-- END accordion -->
-<%= link_to ts('edit.this'), edit_admin_feature_definition_path(parent_object, @definition), class: 'btn btn-primary form-submit' %> |
-<%= link_to ts('cancel.this'), admin_feature_path(parent_object.fid), class: 'btn btn-primary form-submit', id: 'edit-cancel' %>
+<%= link_to ts('cancel.this'), admin_feature_path(parent_object.fid, section: "definitions"), class: 'btn btn-primary form-submit', id: 'edit-cancel' %> |
+ <%= link_to ts('edit.this'), edit_admin_feature_definition_path(parent_object, @definition), class: 'btn btn-primary form-submit' %>
```

Comment:
likewise this should be object instead @ definiton .

Reply:
for what it's worth the view already used @definition - i just reversed the order of the buttons so cancel came first. 

If you look at the entire file, it uses the instance variable @definition for all of the output. Do you want that changed to object?